App/Components/NotificationsBar.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component, PropTypes } from 'react'
+import { View, Modal, Text, TouchableOpacity, Image } from 'react-native'
+import { Images } from '../Themes/'
+import StatusBarAlert from 'react-native-statusbar-alert'
+import PurpleGradient from './PurpleGradient'
+import styles from './Styles/ModalStyle'
+
+const NotificationScreen = ({notifications, onDismissModal}) => {
```

Comment:
What was your reasoning behind not breaking this into a separate component in its own file?